src/theory/quantifiers/sygus/term_database_sygus.h

```diff
@@ -183,6 +183,12 @@ class TermDbSygus {
   /** same as above, but with a cache of visited nodes */
   Node evaluateWithUnfolding(
       Node n, std::unordered_map<Node, Node, NodeHashFunction>& visited);
+  /** is evaluation head?
```

Comment:
well, ``x`` is the evaluation head of ``n`` in this case. The correct name should maybe be ``isEvaluationPoint`` or ``isEvaluationApp``.